flow/designs/nangate45/aes/config.mk

```diff
@@ -7,6 +7,7 @@ export SDC_FILE      = ./designs/$(PLATFORM)/$(DESIGN_NICKNAME)/constraint.sdc
 
 # These values must be multiples of placement site
 # x=0.19 y=1.4
+export FLOORPLAN_DEF = ./designs/$(PLATFORM)/$(DESIGN_NICKNAME)/aes_ng45_fp.def
```

Comment:
Why does this test need an external floorplan?

Reply:
I converted this test to use an external floorplan def to validate the flow.   The external floorplan def was created from 2_2_floorplan.def from the previous flow  -- We will have more scenarios with using an external floorplan DEF (with size and pin and/or macro placements) to initialize the floorplan